allianceutils/checks.py

```diff
@@ -186,7 +204,20 @@ def check_git_hooks(app_configs: Iterable[AppConfig], **kwargs) -> List[CheckMes
         # there is a .git dir:
         #   If dev then there must be a .git/hooks symlink
         #   If in prod then there should be a .git/hooks symlink if there is a .git dir
+
         git_hooks_path = Path(git_path, 'hooks')
+
+        # Check the git config for a hooksPath setting (now used by husky)
+        git_config_result = subprocess.run(
+            ["git", "config", "core.hooksPath"],
+            capture_output=True,
+            cwd=git_path,
+        )
+        if not git_config_result.returncode:
```

Comment:
nitpicky: i think changing this one to returncode == 0 / != 0 is more reader friendly